backend/src/components/audit.js

```diff
@@ -18,17 +18,25 @@ const audit = {
       throw new Error(errMsg);
     }
     try {
-      //const history = await auditService.find(applicationAcronym);
-      const history = [1, 2, 3];
+      const history = await auditService.findHistory(applicationAcronym);
       log.verbose('getHistoryByAcronym', utils.prettyStringify(history));
       if (history) {
-        return history;
+        const hsts = history.Lifecycles.map(lc => {
+          const hst = {};
+          hst.date = lc.createdAt;
+          hst.acronym = history.acronym;
+          hst.user = lc.LifecycleHistories[0].User.username;
+          hst.environment = lc.env;
+          hst.details = lc.appConfig;
+          return hst;
```

Comment:
As this is direct object assignments, it'd be better to write it as the following:
```
return {
  acronym: history.acronym,
  date: lc.createdAt,
  details: lc.appConfig,
  environment: lc.env,
  user: lc.LifecycleHistories[0].User.username
}
```
This is slightly more memory efficient as there's one less object constant being instantiated -> instead the object is just constructed as part of the anonymous function definition directly.

Reply:
Right, good call. Fixed